YAPA/MainWindow.xaml.cs

```diff
@@ -682,8 +672,15 @@ private void ResetTicking()
 
         private void MainWindow_MouseLeftButtonDown(object sender, MouseButtonEventArgs e)
         {
-            base.OnMouseLeftButtonDown(e);
-            DragMove();
+            try
+            {
+                base.OnMouseLeftButtonDown(e);
+                DragMove();
+                e.Handled = true;
+            }
+            catch
```

Comment:
it was a crash issue, the app crashes immediately, I thought set e.Handled to true will resolve this, but it does not.
I don't have a exactly reproduce steps.
in my momory, I just turn on the computer, double click on the YAPA icon on desktop, after the main windows shown, I drag the window to somewhere else, then it crashed.